src/citrine/_rest/resource.py

```diff
@@ -39,3 +42,13 @@ def access_control_dict(self) -> dict:
             "type": self._resource_type.value,
             "id": str(self.uid)
         }
+
+
+class ResourceRef(Serializable['ResourceRef']):
```

Comment:
I originally moved and renamed this class because I was trying to get around circular imports. But I think the move is worth keeping, since this isn't exclusively used for modules anymore.